processing/manual_correction.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Run this script in results/data folder
+#
+# USAGE:
+# Define files.yaml file as following:
+#   ---
+#   FILES_SEG:
+#   - sub-amu01_T1w_RPI_r.nii.gz
+#   - sub-amu01_T2w_RPI_r.nii.gz
+#   - sub-brnoCeitec01_T2star_rms.nii.gz
+#   FILES_LABEL:
+#   - sub-amu01
+#   - sub-brnoUhb01
+#
+#
+# THEN RUN:
+#   PATH_TO_SPINEGENERIC/processing/manual_correction.sh files.yaml
+#
+# Authors: Julien Cohen-Adad, Jan Valosek
+
+
+# function for parsing input yaml file
+yaml_parser() {
+
+  INPUT_YAML=$1
+
+  # tr: delete all newlines begining with dash (-), so convert input file into one line without dashes
+  # (this also deletes all other dashes, so it is necessary to use last sed)
+  # 1st sed: delete "FILES_LABEL:" string and everything after it
+  # 2nd sed: delete "FILES_SEG:" string itself
+  # 3rd sed: replace "sub" by "sub-" (tr command deleted all dashes (-))
+  FILES_SEG=$(cat $INPUT_YAML | tr -d '\n-' | sed 's/FILES_LABEL:.*//' | sed 's/FILES_SEG://' | sed 's/sub/sub-/g')
+  # tr: delete all newlines begining with dash (-), so convert input file into one line without dashes
+  # (this also deletes all other dashes, so it is necessary to use last sed)
+  # 1st sed: delete everything before "FILES_LABEL:" string including string "FILES_LABEL:" itself
+  # 2nd sed: replace "sub" by "sub-" (tr command deleted all dashes (-))
+  FILES_LABEL=$(cat $INPUT_YAML | tr -d '\n-' | sed 's/.*FILES_LABEL://' | sed 's/sub/sub-/g')
+}
+
+# Print help, if invalid input
+if [[ $# != 1 ]] || [[ $1 == "--help" ]] || [[ $1 == "-h" ]];then
+
+  echo -e "Invalid input. \n\nPlease define files.yaml file with list of segmentations to correct"
+  echo -e "and list of subjects to correct labeling on (one file perline)."
```

Comment:
quick search shows that yml is more common
also, replace the "here" by an example. people need examples
```suggestion
  echo -e "Invalid input. \n\nPlease create a files.yml file, which lists the files associated with the segmentations or labels to manually correct"
```
```suggestion
  echo -e "Invalid input. \n\nPlease define files.yaml file with list of segmentations to correct"
  echo -e "and list of subjects to correct. See example HERE"
```